karma.conf.js

```diff
@@ -10,6 +10,30 @@ module.exports = function (config) {
     // Only test IE latest.
     config.browsers.push('internet_explorer_11');
   }
+  var polyOptions = process.argv.filter((val) => {
+    return val.search('--polyfills=') == 0;
```

Comment:
nit: `val.indexOf()` is probably more appropriate because we don't require a regex (also `===`)